src/OrchardCore.Modules/OrchardCore.Email/Workflows/Activities/EmailTask.cs

```diff
@@ -76,7 +76,7 @@ public override async Task<ActivityExecutionResult> ExecuteAsync(WorkflowExecuti
             var sender = await _expressionEvaluator.EvaluateAsync(Sender, workflowContext);
             var recipients = await _expressionEvaluator.EvaluateAsync(Recipients, workflowContext);
             var subject = await _expressionEvaluator.EvaluateAsync(Subject, workflowContext);
-            var body = await _expressionEvaluator.EvaluateAsync(Body, workflowContext);
+            var body = await _expressionEvaluator.EvaluateAsync(Body, workflowContext, System.Text.Encodings.Web.HtmlEncoder.Default);
```

Comment:
Maybe you could inject `HtmlEncoder` that would use the current configured encoder options

Reply:
Done